smartsim/_core/_cli/build.py

```diff
@@ -30,104 +30,38 @@
 import typing as t
 from pathlib import Path
 
-import pkg_resources
 from tabulate import tabulate
 
-from smartsim._core._cli.utils import (color_bool, pip_install,
-                                       SMART_LOGGER_FORMAT)
+from smartsim._core._cli import verify
+from smartsim._core._cli.utils import (
+    SMART_LOGGER_FORMAT,
+    color_bool,
+    pip_install,
+    pip_uninstall,
+)
 from smartsim._core._install import builder
 from smartsim._core._install.buildenv import (
     BuildEnv,
+    DbEngine,
     SetupError,
     Version_,
+    VersionConflictError,
```

Comment:
This is not necessarily a "you" problem, but should these `Error`s actually be moved to the central `error.py`?